### DISCUSSION TITLE
Problem with the captcha ⚠️

### DISCUSSION THREAD

**Discussion by SebastienXia** [community/community#111543] [Other Feature Feedback, Questions, & Ideas]
![captcha](https://kinsta.com/fr/wp-content/uploads/sites/4/2019/10/login-no-captcha.png)

# Hello everyone 👋

First of all, I would like to report the abusive spam abuse by users named **_@JaneBrowns_** & **_@zhaidul00_** and others **This permanently disrupts community help between users**.

![profile_id_1](https://github.com/community/community/assets/105004912/867e9384-6408-4c20-9000-b9891b7a35a6)
![profile_id_2](https://github.com/community/community/assets/105004912/0035bc1f-6515-4c78-a1f1-8a064e23bd81)

> I've prove but you can also look in the community with their spam...

## The problem with the captcha

I wanted to **report** these two people for their abusive spam but when I finished the small form [here](https://support.github.com/contact/report-abuse?category=report-content&report=JaneBrowns&report_content_url=https%3A%2F%2Fgithub.com%2Fcommunity%2Fcommunity%2Fdiscussions%2F111504&report_id=157332659&report_type=content), github asked me to do a Captcha **8 times which is a long time in fact** and more after having finished the captcha, and press on the "send report" button, I get this message:

![nulllllle](https://github.com/community/community/assets/105004912/34977395-8146-47eb-bf32-6f400d8e2f61)

> I have tried to do the reports several times but each time it gives me this message

https://github.com/community/community/assets/105004912/eb039680-c3eb-4342-ab69-0309cb7f6344

### How can we have a means of reporting without being able to do so because of a captcha that doesn't work very well ?

`Thanks for reading and hope this fixes the problem with the captcha`

**Comment by samus-aran** [comment]
Thank you @SebastienXia for raising this. We've passed this along to our internal teams to investigate this. 

The best route to report users in the community (specifically this community) you can reach out to the GitHub team using our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). We also receive reports if you flag particular discussions or comments within repo (community).

Lastly I've gone ahead and moved this discussion into a more appropriate category for you. 👍

**Reply by shape55** [reply]
Can you please provide an update on this?  Did you see my reply below?

**Comment by shape55** [comment]
Hi, but this still hasn't been fixed.  It's a bug.  I even tried to submit this as a support ticket, and it gives me the same invalid captcha error even when there is no captcha on the page!  I also tried in 2 different browsers (Safari & Firefox).

I tried to submit this message below using this link: https://support.github.com/contact/account w/ the subject "New Captchas are not working", and I still get the same error below.  These forms were submitting just fine before this recent change.

-------
When I try to report spam using [this link](https://support.github.com/contact/report-abuse?category=report-content&report=Shainevizada&report_content_url=https%3A%2F%2Fgist.github.com%2F%2Fcoolaj86%2F6f4f7b30129b0251f61fa7baaa881516%3Fpermalink_comment_id%3D4999397%23gistcomment-4999397&report_id=164776779&report_type=content), I fill out the form as I’ve done over 20 times before, complete the new captcha, **get the green check mark to confirm it was correct**, then click the “Send request” button, but I then get this error below at this link: https://support.github.com/contact/report-abuse

> Sorry, the captcha result was invalid. Please visit https://docs.github.com/articles/troubleshooting-connectivity-problems/#troubleshooting-the-captcha for troubleshooting information.
> 
> So that we can accurately identify the user or content you wish to report please return to GitHub and follow the instructions in the Reporting Abuse or Spam article.
